client/webpack-workers.js

```diff
@@ -8,6 +8,7 @@ module.exports = (env = {}) => {
   const staticPath = path.join(__dirname, './static/js');
   const configPath = path.join(__dirname, '../config/client');
   return {
+    cache: __DEV__ ? { type: 'filesystem' } : false,
```

Comment:
If we have no additional config for `cache`, why are we not using `type: 'memory'`? https://webpack.js.org/configuration/other-options/#cachetype

_You will probably need to explain like I am 5_

Reply:
To me, it looks like, that way the cache persists between hard restarts of the development cycles?

Reply:
From my understanding, it literally uses RAM.  So, yeah, it won't persist.